x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -704,3 +717,22 @@ export const ProcessEventDot = styled(ProcessEventDotComponents)`
     color: white;
   }
 `;
+
+const processTypeToCube: Record<ResolverProcessType, keyof NodeStyleMap> = {
```

Comment:
:red_circle: This and `nodeType` below got moved to `/view/assets` on master. Did you intend to duplicate them here?